person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name, age = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_service?
+    is_of_age || parent_permission
+  end
```

Comment:
- [ ] The `of_age?` method should be a private method.

- [ ] For the `can_use_services?` method you should have `of_age?`, not `is_of_age`. And you should use the instance `@parent_permission` instead of `parent_permission`, which would throw errors.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
  ```